cupyx/signal/convolution/__init__.py

```diff
@@ -0,0 +1 @@
+from cupyx.signal.convolution._convolve import convolve1d3o  # NOQA
```

Comment:
Would it be better to avoid introducing a new public namespace? How about naming it `_convolution`?

I think cuSignal's public APIs are defined at `cusignal.*` so we only need a one namespace `cupyx.signal.*` to host for them.

Reply:
It makes sense. I changed it as such.